src/views/MultiTimeseries/MultiTimeseriesView.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent, useMemo } from 'react';
+import { MultiTimeseriesViewData } from './MultiTimeseriesViewData';
+import ViewWrapper from './ViewWrapper';
+
+type Props = {
+    data: MultiTimeseriesViewData
+    width: number
+    height: number
+}
+
+const MultiTimeseriesView: FunctionComponent<Props> = ({data, width, height}) => {
+    const divStyle: React.CSSProperties = useMemo(() => ({
+        width,
+        height,
+        overflowX: 'hidden',
+        overflowY: 'auto'
+    }), [width, height])
+
+    const H = height / data.panels.length
```

Comment:
Does this need to take into account the relative-height?

Reply:
yes it does. not implemented yet. Let's do that in a separate PR